### PR TITLE
Component fixes

### PR DESCRIPTION
**Pull request by ernzher**
DoenetML components fixes (make inline) - updateValue, callAction, triggerSet 
Refactor general components - Checkbox, Button, IncrementMenu
uiDocs - Checkbox and some minor fixes

### CODE REVIEW

File: src/_reactComponents/PanelHeaderComponents/Button.jsx [hunk 1]
```diff
@@ -1,54 +1,49 @@
-import React, { useState } from 'react';
```
Comment (clary149) [inline src/_reactComponents/PanelHeaderComponents/Button.jsx:1]: What task are you completing in this file? Creates weird right margin for buttons used in menus.
Screen Shot 2022-05-15 at 1 32 06 PM

File: src/_reactComponents/PanelHeaderComponents/Checkbox.jsx [hunk 2]
```diff
@@ -6,48 +6,52 @@ import {
 import React, { useRef } from 'react';
```
Comment (clary149) [inline src/_reactComponents/PanelHeaderComponents/Checkbox.jsx:6]: Also creates weird margin on right side of component.
Screen Shot 2022-05-15 at 1 33 26 PM
Screen Shot 2022-05-15 at 1 35 30 PM

**Review by clary149 — CHANGES_REQUESTED** [review]
See doenet.org for how margins should look

Still needs IncrementMenu centered

**Comment by kevincharles** [comment]
It's my understanding these changes are in other PR's.  closing

**Pull request closed without merging**